src/server/scripts/Commands/cs_gobject.cpp

```diff
@@ -38,6 +38,11 @@ EndScriptData */
 #include "RBAC.h"
 #include "WorldSession.h"
 
+struct npc_commandscript // partial declaration so we can map the spawn group command to both .npc and .gobject
```

Comment:
C++ does not support partial class/struct definitions, you are violating one definition rule. http://en.cppreference.com/w/cpp/language/definition

This would only be ok if npc_commandscript was a namespace